front/components/vaults/VaultDataSourceViewContentList.tsx

```diff
@@ -170,8 +206,31 @@ export const VaultDataSourceViewContentList = ({
     owner,
     parentId,
     pagination,
-    viewType: isValidContentNodesViewType(viewType) ? viewType : "documents",
+    viewType: isValidContentNodesViewType(viewType) ? viewType : undefined,
+  });
+
+  const { hasContent: hasDocuments } = useStaticDataSourceViewHasContent({
+    owner,
+    dataSourceView,
+    parentId,
+    viewType: "documents",
   });
+  const { hasContent: hasTables } = useStaticDataSourceViewHasContent({
```

Comment:
I guess we can do only one call for the viewType not currently rendered and rely on the previous call to `useDataSourceViewContentNodes`, WDYT?

Reply:
Not really, because I want to hide the dropdown of the other view type (and not re-do the query every time you switch). Alternative is to just re-do the query on every switch. Do you think it's preferable ?

Reply:
Ok let's keep like this and see if it's not too heavy. Given that we only do it for static it should be fine.